unit/models/check_deposit.py

```diff
@@ -1,6 +1,10 @@
 import json
 from unit.utils import date_utils
-from typing import Optional, IO, Literal
+try:
+    from typing import Optional, IO
```

Comment:
@henriquesalvaro I think you forgot Literal here.
```suggestion
    from typing import Optional, IO, Literal
```